integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java

```diff
@@ -89,8 +84,14 @@ public void setMetalakeCommentField(String commentField) {
   }
 
   public void setQueryInput(String queryInput) {
```

Comment:
I think better change function and variable name `QueryInput` to `QueryParams`.

Reply:
Done